Example/Source/Examples/Collections/RepeatCollectionBrickViewController.swift

```diff
@@ -67,9 +67,9 @@ class RepeatCollectionBrickViewController: BrickApp.BaseBrickController, BrickRe
 
 extension RepeatCollectionBrickViewController: CollectionBrickCellDataSource {
 
-    func configureCollectionBrickViewForBrickCollectionCell(brickCollectionCell: CollectionBrickCell) {
-        brickCollectionCell.brickCollectionView.registerBrickClass(LabelBrick.self)
-        brickCollectionCell.brickCollectionView.registerBrickClass(ImageBrick.self)
+    func registerBricks(for cell: CollectionBrickCell) {
```

Comment:
As you are registering just the class, shouldn't you add these brickTypes in the initializer of the CollectionBrick?